src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -80,6 +79,25 @@ G'.adj_sub h
 /-- A subgraph is called a *spanning subgraph* if it contains all the vertices of `G`. --/
 def is_spanning (G' : subgraph G) : Prop := ∀ (v : V), v ∈ G'.verts
 
+/-- Coercion from `subgraph G` to `simple_graph V`. This will include "junk"
+isolated vertices unless the subgraph is spanning, hence the name. -/
```

Comment:
```suggestion
/-- Coercion from `subgraph G` to `simple_graph V`.  If `G'` is a spanning
subgraph, then `G'.spanning_coe` yields an isomorphic graph.
In general, this adds in all vertices from `V` as isolated vertices. -/
```
I hesitate calling the vertices junk because you might to create a spanning subgraph this way.